src/logics/syncswap/logic.swap-token.test.ts

```diff
@@ -0,0 +1,119 @@
+import { LogicTestCase } from 'test/types';
+import { Router__factory } from './contracts';
+import { SwapTokenLogic, SwapTokenLogicFields, SwapTokenLogicOptions } from './logic.swap-token';
+import * as common from '@protocolink/common';
+import { constants, utils } from 'ethers';
+import * as core from '@protocolink/core';
+import { expect } from 'chai';
+import { getContractAddress } from './configs';
+import { zksyncTokens } from './tokens';
+
+describe('SyncSwap SwapTokenLogic', function () {
+  context('Test getTokenList', async function () {
+    SwapTokenLogic.supportedChainIds.forEach((chainId) => {
+      it(`network: ${common.toNetworkId(chainId)}`, async function () {
+        const logic = new SwapTokenLogic(chainId);
+        const tokenList = await logic.getTokenList();
+        expect(tokenList).to.have.lengthOf.above(0);
+      });
+    });
+  });
+
+  context('Test build', function () {
```

Comment:
The test file does not have `'Test quote'`. Is it because we cannot fork from mainnet in local env.?

Reply:
[Oral Discussion]